js/ui/status/missionGame.js

```diff
@@ -0,0 +1,648 @@
+        this._inventory = createSubMenuMenuItemWithFauxParent(_('Inventory'), parentMenu);
+
+        /* Add switches and toolbox items to hbox */
+        this.add(missionSwitch.actor);
+        addSubMenuItemToBox(this._adventures, this, parentMenu);
+        addSubMenuItemToBox(this._spells, this, parentMenu);
+        addSubMenuItemToBox(this._inventory, this, parentMenu);
+
+        /* When we get some new adventures or spells, add the menu items to the list again */
+        this._service.connect('discover-new-adventures', Lang.bind(this, function(chat, adventures) {
+            this._adventures.menu.removeAll();
+            adventures.forEach(Lang.bind(this, function(adventure) {
+                this._adventures.menu.addAction(adventure.desc, launchLessonAction(adventure.name));
+            }));
+        }));
+        this._service.connect('discover-new-spells', Lang.bind(this, function(chat, spells) {
+            this._spells.menu.removeAll();
+            spells.forEach(Lang.bind(this, function(spell) {
+                this._spells.menu.addAction(spell.desc, launchLessonAction(spell.name));
+            }));
+        }));
+        this._service.connect('discover-new-inventory-items', Lang.bind(this, function(chat, items) {
+            this._inventory.menu.removeAll();
+            items.forEach(Lang.bind(this, function(item) {
+                this._inventory.menu.addAction(item.name, function() {
+                });
+            }));
+        }));
+    }
+});
+
+function createSeparator() {
+    let separator = new St.DrawingArea({ style_class: 'calendar-vertical-separator',
+                                           pseudo_class: 'highlighted' });
+    separator.connect('repaint', Lang.bind(this, function(area) {
+        let cr = area.get_context();
+        let themeNode = area.get_theme_node();
+        let [width, height] = area.get_surface_size();
+        let stippleColor = themeNode.get_color('-stipple-color');
+        let stippleWidth = themeNode.get_length('-stipple-width');
+        let x = Math.floor(width/2) + 0.5;
+        cr.moveTo(x, 0);
+        cr.lineTo(x, height);
+        Clutter.cairo_set_source_color(cr, stippleColor);
+        cr.setDash([1, 3], 1); // Hard-code for now
+        cr.setLineWidth(stippleWidth);
+        cr.stroke();
+        cr.$dispose();
+    }));
+
+    return separator;
+}
+
+/* Separate indicators option */
+const MissionGameToolboxIndicator = new Lang.Class({
+    Name: 'MissionGameToolboxIndicator',
+    Extends: PanelMenu.SystemStatusButton,
+
+    _init: function() {
+        this.parent('starred-symbolic', _('Mission'));
+        this.setIcon('starred-symbolic');
+
+        this._service = MissionGameService.getService();
+
+        /* Create layout for indicator menu */
+        let hbox = new St.BoxLayout({name: 'menuArea'});
+
+        /* Add toolbox, separator, chatbox */
+        hbox.add(new MissionToolbox({}, this.menu, this._service));
```

Comment:
They're options for testing based on @dalio48 's feedback. Option 1 is to have a toolbox and a chatbox indicator, and then option 2 is to have them both merged into one. I probably should have added a comment about that.